### PR TITLE
Disabled integrations

### LINKED ISSUE #1135: Telegram service should not delete integration, but inform user about expiration of token, etc.
Telegramm service should not delete integration, but inform user about expiration of token, etc.

### PR DESCRIPTION
**Pull request by BykhovDenis**
close #1135 Signed-off-by: Denis Bykhov [80476319+BykhovDenis@users.noreply.github.com](mailto:80476319+BykhovDenis@users.noreply.github.com)

### CODE REVIEW

**Comment by BykhovDenis** [comment]
We need also update telegram service after bump '@anticrm/setting'

File: plugins/setting-assets/lang/en.json [hunk 1]
```diff
@@ -25,6 +25,9 @@
     "Settings": "Settings",
     "SelectWorkspace": "Select workspace",
     "DeleteStatus": "Delete status",
-    "DeleteStatusConfirm": "Do you want to delete this status?"
+    "DeleteStatusConfirm": "Do you want to delete this status?",
+    "Reconnect": "Reconnect",
+    "IntegrationDisabled": " has been disabled",
```
Comment (haiodo) [inline plugins/setting-assets/lang/en.json:30]: Why not use {substitution} for word translation?

File: plugins/setting-resources/src/components/activity/TxIntegrationDisable.svelte [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  import { TxUpdateDoc } from '@anticrm/core'
+  import { IntlString } from '@anticrm/platform'
+  import { getClient } from '@anticrm/presentation'
+  import { Integration } from '@anticrm/setting'
+  import { Label } from '@anticrm/ui'
+  import setting from '../../plugin'
+
+  export let tx: TxUpdateDoc<Integration>
+  export let doc: Integration
+  $: console.log(doc)
```
Comment (haiodo) [inline plugins/setting-resources/src/components/activity/TxIntegrationDisable.svelte]: Extra logging

File: plugins/setting-resources/src/components/activity/TxIntegrationDisable.svelte [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+  async function getTypeLabel (tx: TxUpdateDoc<Integration>): Promise<IntlString | undefined> {
+    const doc = await client.findOne(setting.class.Integration, { _id: tx.objectId })
+    if (doc === undefined) return
+    const type = await client.findOne(setting.class.IntegrationType, { _id: doc.type })
+    return type?.label
+  }
+</script>
+
+&nbsp;
```
Comment (haiodo) [inline plugins/setting-resources/src/components/activity/TxIntegrationDisable.svelte:36]: Does it required?

File: plugins/telegram-resources/src/components/Reconnect.svelte [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+<script lang="ts">
+  import { getMetadata, serialize } from '@anticrm/platform'
+  import { Button, EditBox, IconClose, Label } from '@anticrm/ui'
+  import { createEventDispatcher } from 'svelte'
+  import login from '@anticrm/login'
+  import PinPad from './PinPad.svelte'
+  import telegram from '../plugin'
+  import { getClient } from '@anticrm/presentation'
+  import setting from '@anticrm/setting'
+import { getCurrentAccount, Ref, Space } from '@anticrm/core';
```
Comment (haiodo) [inline plugins/telegram-resources/src/components/Reconnect.svelte]: Indent

**Review by haiodo — APPROVED** [review]

File: plugins/setting-resources/src/components/activity/TxIntegrationDisable.svelte (hunk 3 above)
Comment (BykhovDenis) [reply plugins/setting-resources/src/components/activity/TxIntegrationDisable.svelte:36]: Yes(, without it no spaces between words

File: plugins/setting-assets/lang/en.json (hunk 1 above)
Comment (BykhovDenis) [reply plugins/setting-assets/lang/en.json:30]: Because integration label is IntlString, and we must get and translate it before

**Pull request merged**